wandb/filesync/dir_watcher.py

```diff
@@ -291,18 +299,14 @@ def _get_file_event_handler(
                 )
             else:
                 make_handler = PolicyEnd
-                for policy, globs in self._user_file_policies.items():
-                    if policy == "end":
-                        continue
+                for policy, handler_type in [("now", PolicyNow), ("live", PolicyLive)]:
```

Comment:
minor suggestion: maybe we can add the policy name as a class attribute of the different policies?
i.e.
```
class PolicyNow:
  name = "now"
```
or something like this? what do you think?

Reply:
I like this idea and I will implement it in a followup PR, if that's okay! Just to keep this PR maximally-obviously-safe.